README.md

```diff
@@ -135,6 +135,20 @@ git, I worked on github and gitlab. I tried using git guis but I prefer to work
 
 * Explain event delegation
 * Explain how `this` works in JavaScript
+    This points to an object depending on the context,
+    In what function was used in which we defined it.
```

Comment:
this points to an object depending on the function context in which it was declared